content/applications/hr/payroll/reporting.rst

```diff
@@ -0,0 +1,205 @@
+   :align: center
+   :alt: Send the data to a spreadsheet.
+
+Once in the spreadsheet view, click on :guilabel:`File`, then select :guilabel:`Save` to save the
+data, or :guilabel:`Save as Template` to use the current template in the future.
+
+Meal Vouchers
+-------------
+
+:guilabel:`Meal Vouchers` provides an overview of  the meal vouchers used by employees, and can be
+shown by day, week, month, quarter, or year. The default view is by month.
+
+.. image:: reporting/meal-vouchers.png
+   :align: center
+   :alt: Meal voucher
+
+To change the displayed view, click on :guilabel:`Total`. The data will collapse, showing only the
+count column. Click on :guilabel:`Total` again, then hover over *Day*, then click on one of the
+other time-period options available.
+
+.. image:: reporting/voucher-day.png
+   :align: center
+   :alt: Meal voucher
+
+It is possible to compare the meal voucher report to the previous time period currently displayed,
+or the previous year. To view these comparisons, click on the on the :guilabel:`Comparison` drop
+down at the top, then select either :guilabel:`Date: Previous Period` or
+:guilabel:`Date: Previous Year`.
+
+.. image:: reporting/meal-comparison.png
+   :align: center
+   :alt: Meal voucher
+
+To export the data in an xlsx format, click the :guilabel:`Download xlsx` icon. The information will
+be downloaded into an Excel spreadsheet.
+
+.. image:: reporting/voucher-download.png
+   :align: center
+   :alt: Download an Excel spreadsheet of the data by clicking the download button.
+
+The data can be inserted in a spreadsheet by clicking the :guilabel:`Insert in Spreadsheet` smart
+button. A pop up will appear asking which spreadsheet to place the information. Select an existing
+spreadsheet or enter the name for a new spreadsheet. Click the :guilabel:`Confirm` smart button to
+move to a spreadsheet view with all the information populated.
+
+Once in the spreadsheet view, click on :guilabel:`File`, then select :guilabel:`Save` to save the
+data, or :guilabel:`Save as Template` to use the current template in the future.
+
+Attachment of Salary
+--------------------
+
+The :guilabel:`Attachment of Salary` report shows all deductions or allocations per employee,
+such as child support payments and wage garnishments.
+
+.. image:: reporting/attachment-of-salary.png
+   :align: center
+   :alt: View the Attachment of Salary report that shows all salary garnishments.
+
+The employees are listed in the left side column, while the different deductions are listed in the
+top row, organized by month.
+
+The report can be exported as a xlsx file, or inserted into a spreadsheet, using the corresponding
+smart buttons at the top.
+
+Clicking on the :guilabel:`Measures` smart button wil show the options of
+data to display. Assignment of salary, attachment of salary, child support, and count, can all be
+selected or deselected by clicking on the item. If an item has a check mark, it will be displayed.
```

Comment:
```suggestion
Clicking on the :guilabel:`Measures` smart button will show options for
how the data is displayed. Assignment of salary, attachment of salary, child support, and count can all be
selected or deselected by clicking on the item. If an item has a check mark, it will be displayed.
```